.github/workflows/perl-lint-checks.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     name: "Perltidy"
     container:
-      image: perldocker/perl-tester
+      image: registry.opensuse.org/devel/openqa/containers/os-autoinst_dev
```

Comment:
just for clarity, i see two images in the registry. 
`devel/openqa/containers/opensuse/os-autoinst-dev `
`devel/openqa/containers/os-autoinst_dev`

What s the difference?

Reply:
We don't know :shrug:

Reply:
actually have the same hash. i guess ignore.